spring-cloud-dataflow-admin-starter/src/main/java/org/springframework/cloud/dataflow/admin/AdminApplication.java

```diff
@@ -16,31 +16,21 @@
 
 package org.springframework.cloud.dataflow.admin;
 
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.List;
-
 import org.springframework.boot.SpringApplication;
-import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
-import org.springframework.boot.autoconfigure.SpringBootApplication;
-import org.springframework.boot.autoconfigure.security.oauth2.OAuth2AutoConfiguration;
+import org.springframework.context.annotation.Configuration;
 
 /**
  * Bootstrap class for Spring Cloud Data Flow Admin.
  *
  * @author Mark Fisher
+ * @author Josh Long
  */
-@SpringBootApplication
-@EnableAutoConfiguration(exclude = {
-		OAuth2AutoConfiguration.class
-})
+@Configuration
+@EnableDataflowAdminServer
```

Comment:
I find it interesting to remove `@SpringBootApplication` on this **starter** class while the `@EnableDataflowAdminServer` doesn't have `@SpringBootApplication` either. But I see 'EnableAutoConfiguration` in `AdminConfiguration` class. Is it worth to annotate  `@EnableDataflowAdminServer` with `@SpringBootApplication` and remove `@EnableAutoConfiguration` from `AdminConfiguration` (which could make all the required annotation in one place and leaving the AdminConfiguration as just a Configuration class)?